src/event.py

```diff
@@ -7,25 +7,37 @@
 from dateutil import parser
 
 
+# import regular expression module
+import re
+
+
+
 class Event:
     """Event records all the data from an event, and has methods to generate the
     message from an event
     """
 
     # pylint: disable=too-many-instance-attributes
     # Events have lots of data that we need to save together
+    
+
 
     def __init__(self, title, group_name, description, location, time, url, status, uuid):
         # pylint: disable=too-many-arguments
         self.title = title
         self.group_name = group_name
-        self.description = description
+        self.description = self.remove_html(description)
         self.location = location
         self.time = time
         self.url = url
         self.status = status
         self.uuid = uuid
 
+    def remove_html(self, string):
```

Comment:
```suggestion
        self.description = self.__remove_html(description)
        self.location = location
        self.time = time
        self.url = url
        self.status = status
        self.uuid = uuid

    def __remove_html(self, string):
```

:wrench: Since this function is only intended to be used in `Event`'s `__init__` constructor, we'll want to add two underscores in order to mark this as a private function that isn't intended to be used by other people. But otherwise, separating the HTML sanitation into a separate function is a great approach!